third-party/realdds/src/dds-metadata-syncer.cpp

```diff
@@ -8,73 +8,104 @@
 namespace realdds {
 
 
+dds_metadata_syncer::dds_metadata_syncer() :
+    _is_alive( std::make_shared< bool >( true ) )
+{
+}
+
+
+dds_metadata_syncer::~dds_metadata_syncer()
+{
+    *_is_alive = false;
```

Comment:
Why not just `reset()` it?

Reply:
I gives me an indication if we are in the process of being destructed, but it is a bit redundant. I will remove it (and all related checks)